python/CHANGELOG.md

```diff
@@ -5,6 +5,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.13.9] (2024-06-19)
+[0.13.9]: https://github.com/trezor/trezor-firmware/compare/python/v0.13.8...python/v0.13.9
+
+### Added
+- trezorctl: Automatically go to bootloader when upgrading firmware  [#2919]
+- Support interaction-less upgrade  [#2919]
+- Added user adjustable brightness setting.  [#3208]
```

Comment:
Nit: it would be nice to have period at the end consistenly, before it's automated #3973